WBooks/src/redux/login/actions.js

```diff
@@ -0,0 +1,39 @@
+import { loginApp } from '@services/login';
+
+import { LOGIN_TYPES } from './constants/loginTypes';
+
+export function loginHasErrored(bool) {
```

Comment:
In Wolox we work, with the following pattern for the actions file. We have the actions we expose to other files called `actionCreators` and the action we use inside this file called `privateActionCreators`.
This is how both types of action should be displayed:

```js
const privateActionCreators = {
  login: () => {
  ...
  },
  loginSuccess: data => {
  ...
  },
  loginFailure: error => {
  ...
  }
};

export const actionCreators: {
  login: () => {
    return async dispatch => {
    ...
    }
  }
}
```